Casks/l/lg-onscreen-control.rb

```diff
@@ -8,8 +8,25 @@
   desc "Displays all connected LG monitor information"
   homepage "https://www.lg.com/us/support/monitors"
 
+  # There is no page available specifically for the software
+  # so we return the downloads from one of the popular products
   livecheck do
-    skip "No version information available"
+    url "https://www.lg.com/us/support/product/lg-27GN950-B.AUS"
+    regex(/Mac[._-]OSC[._-]v?(\d+(?:\.\d+)+)\.zip/)
+    strategy :page_match do |page, regex|
+      json_string = page&.[](/NEXT[._-]DATA[^>]*>\s*([^<]+)\s*</i, 1)
+      next if json_string.blank?
+
+      json = Homebrew::Livecheck::Strategy::Json.parse_json(json_string)
+      json.dig("props", "pageProps", "softwareData", "fileData")&.map do |_, files|
+        files.map do |file|
+          match = file["originalFileName"]&.match(regex)
+          next if match.blank?
+
+          "#{match[1]},#{file["fileName"]}"
```

Comment:
```suggestion
          next if match.blank? || (filename = file["fileName"]).blank?

          "#{match[1]},#{filename}"
```

This would return a version like `1.2.3,` if the `fileName` value isn't present, so we should guard against that.